editoast/src/models/timetable.rs

```diff
@@ -92,19 +112,36 @@ impl Timetable {
                         .filter(|stop| stop.duration > 0.)
                         .count() as i64;
 
+                    let rolling_stock = LightRollingStockModel::retrieve_conn(
+                        &mut conn,
+                        train_schedule.rolling_stock_id,
+                    )
+                    .unwrap();
+                    let rolling_stock_version = rolling_stock.unwrap().version;
+                    let update_required = train_validation(
+                        &train_schedule.infra_version.clone().unwrap(),
+                        train_schedule.clone().rollingstock_version.unwrap(),
```

Comment:
Why don't you `clone()` the `rollingstock_version` instead of the`train_schedule` ?